two_factor/templates/two_factor/admin/login.html

```diff
@@ -30,9 +30,6 @@
   <form action="" method="post">{% csrf_token %}
     {% include "two_factor/_wizard_forms.html" %}
 
-    {# hidden submit button to enable [enter] key #}
-    <input type="submit" value="" class="d-none" />
-
```

Comment:
In this template you remove this button, but in all other templates you hide it. What is the reason of this difference?